### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by gaspar-chilingarov**
- Rename G Suite -> Google Workspace
- Add use case for deployment under specific folder

### CODE REVIEW

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR @gaspar-chilingarov 
Looks like this was based off of the main branch, could you base off of `develop` branch?

**Review by rjerrems — APPROVED** [review]
LGTM

File: 0-bootstrap/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The purpose of this step is to bootstrap a GCP organization, creating all the re
 
 1. A GCP [Organization](https://cloud.google.com/resource-manager/docs/creating-managing-organization)
 1. A GCP [Billing Account](https://cloud.google.com/billing/docs/how-to/manage-billing-account)
-1. Cloud Identity / G Suite groups for organization and billing admins
+1. Cloud Identity / Google Workspace groups for organization and billing admins
```
Comment (bharathkkb) [inline 0-bootstrap/README.md:9]: We are working with a tech editor who suggested this in the docs
```suggestion
1. Cloud Identity / Google Workspace  (formerly G Suite) groups for organization and billing admins
```

**Review by bharathkkb — APPROVED** [review]
LGTM, minor suggestion

**Comment by gaspar-chilingarov** [comment]
more changes to update the documentation

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**